### PR TITLE
updates to reflect enterprise features

### PR DESCRIPTION
**Pull request by Daniellem97**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- [x] You have updated the navigation files correctly:
    - [x] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/stack/drift-detection.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Drift detection
 
+!!! info
+    Note that, at least currently, drift detection only works on private workers.
```
Comment (jmfontaine) [inline docs/concepts/stack/drift-detection.md:4]: I would remove "at least currently." I do not want to even look like we are committing to changing that any time soon.

File: docs/integrations/single-sign-on/aws-iam-identity-saml-setup-guide.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # AWS IAM Identity Center SAML Setup Guide
 
+!!! info
+    Note that, at least currently, SSO Integration with SAML 2.0 is an enterprise feature.
```
Comment (jmfontaine) [inline docs/integrations/single-sign-on/aws-iam-identity-saml-setup-guide.md:4]: I would remove "at least currently". I do not want to even look like we are committing to changing that any time soon.

File: docs/integrations/audit-trail.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Audit trail
 
+!!! info
+    Note that, the audit trail is an enterprise feature.
```
Comment (jmfontaine) [inline docs/integrations/audit-trail.md:4]: You are the English-native speaker but is the comma necessary here?

**Review by jmfontaine — CHANGES_REQUESTED** [review]
I could identify at least two more features that are Enterprise plan only:

- Advanced Scheduling
- Private Source Control System

**Comment by Daniellem97** [comment]
Changes made

File: docs/integrations/single-sign-on/aws-iam-identity-saml-setup-guide.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # AWS IAM Identity Center SAML Setup Guide
 
+!!! info
+    Note that SSO Integration with SAML 2.0 is an enterprise feature.
```
Comment (jmfontaine) [inline docs/integrations/single-sign-on/aws-iam-identity-saml-setup-guide.md:4]:
```suggestion
    Note that SSO Integration with SAML 2.0 is an Enterprise plan feature.
```

**Review by jmfontaine — COMMENTED** [review]
Does saying "Enterprise plan" sounds better than "Enterprise"?

**Comment by Daniellem97** [comment]
Agreed, committed changes

**Review by jmfontaine — APPROVED** [review]

**Pull request merged**